.github/workflows/community-review.sh

```diff
@@ -352,6 +415,37 @@ call_github_graphql_api() {
     "https://api.github.com/graphql"
 }
 
+# =============================================================================
+# Check if a label exists in the repository and create it if it doesn't
+# Arguments:
+#   $1 - GitHub API token for authentication
+#   $2 - Label name to check/create
+# =============================================================================
+ensure_label_exists() {
+  local token="${1?missing token}"
+  local label_name="${2?missing label name}"
+  
+  local color="90EE90"
```

Comment:
I would suggest a different color for it so it will be obvious that this is different than the review labels. Something purplish, I checked the existing label and purple is not occupied for anything ATM, so for example `5d069e` would be nice.